src/scripts/OSFramework/OSUI/Behaviors/FocusManager.ts

```diff
@@ -0,0 +1,57 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.OSUI.Behaviors {
+	/**
+	 * Class to manage focus of overlay components
+	 *
+	 * @export
+	 * @class FocusManager
+	 */
+	export class FocusManager {
+		// Store the last focused element
+		private _lastFocusedElem: HTMLElement;
+
+		constructor() {
+			// do nothing.
+		}
+
+		/**
+		 * Method to return the focus to the previous focused element
+		 *
+		 * @return {*}  {void}
+		 * @memberof OSFramework.Behaviors.FocusManager
+		 */
+		public returnFocusToElement(): void {
```

Comment:
@OS-giulianasilva Having a method name starting with "return" is a bit strange (and not even a get since it doesn't return anything). We could have something like "focusOnPreviousElement" or something better instead.

Reply:
done!